### PR TITLE
Pull request for WAZO-891-line-application

### PR DESCRIPTION
**Pull request by fblackburn1**
gitignore: add egg-info
pjsip: convert application to context

### CODE REVIEW

File: xivo_confgen/plugins/pjsip_conf.py [hunk 1]
```diff
@@ -276,6 +276,9 @@ def _get_user_endpoint(self, user_sip, pickup_groups):
         for key, value in all_options:
             user_dict[key] = value
 
+        if user_sip.application_uuid:
+            user_dict['context'] = 'wazo-app-{}'.format(user_sip.application_uuid)
```
Comment (pc-m) [inline xivo_confgen/plugins/pjsip_conf.py]: I think this should be `stasis-wazo-app-{}`
https://blogs.asterisk.org/2019/03/27/stasis-improvements-goodbye-dialplan/

Comment (fblackburn1) [reply xivo_confgen/plugins/pjsip_conf.py]: Yes you can make a PR if you want. Don't forget to change wazo-calld too
https://github.com/wazo-pbx/wazo-calld/blob/master/wazo_calld/plugins/applications/stasis.py#L20

Comment (fblackburn1) [reply xivo_confgen/plugins/pjsip_conf.py]: Did you test it?

Comment (pc-m) [reply xivo_confgen/plugins/pjsip_conf.py]: No I was just trying to remember the prefix that I needed to use for my calld test and noticed that there was no prefix here.

I used `dialplan show stasis-<TAB>`  to confirm the auto-generated context name

Comment (fblackburn1) [reply xivo_confgen/plugins/pjsip_conf.py]: fixed

**Pull request merged**